lib/srv/regular/sshserver.go

```diff
@@ -1601,6 +1601,8 @@ func (s *Server) dispatch(ctx context.Context, ch ssh.Channel, req *ssh.Request,
 		case teleport.ForceTerminateRequest:
 			return s.termHandlers.HandleForceTerminate(ch, req, serverContext)
 		case sshutils.EnvRequest, tracessh.EnvsRequest:
+		case sshutils.FileTransferRequestResponse:
+			return s.termHandlers.HandleFileTransferRequestResponse(ctx, ch, req, serverContext)
```

Comment:
Is this handling a request, a response, or both? The name makes it hard to tell.